src/Workspaces/CSharp/Portable/CSharpWorkspaceResources.resx

```diff
@@ -144,4 +144,28 @@
   <data name="Implement_Interface" xml:space="preserve">
     <value>Implement Interface</value>
   </data>
+  <data name="Code_block_preferences" xml:space="preserve">
+    <value>Code-block preferences</value>
+  </data>
+  <data name="Expression_bodied_members" xml:space="preserve">
+    <value>Expression-bodied members</value>
+  </data>
+  <data name="Indentation_preferences" xml:space="preserve">
+    <value>Indentation preferences</value>
+  </data>
+  <data name="Null_checking_preferences" xml:space="preserve">
+    <value>Null-checking preferences</value>
+  </data>
+  <data name="Pattern_matching_preferences" xml:space="preserve">
+    <value>Pattern matching preferences</value>
```

Comment:
Pattern-matching?  to match the above naming patterns?

Reply:
The naming is directly coming from the sample on docs.microsoft.com. Same for the comments below - @cyrus.najmabadi@gmail.com we can probably consider filing a doc bug to finalize the terminology for our documentation, and I can submit a follow-up PR to update these strings once the documentation bug is fixed. Sounds reasonable?

---
In reply to: [219241244](https://github.com/dotnet/roslyn/pull/30043#discussion_r219241244) [](ancestors = 219241244)

Reply:
[https://github.com/MicrosoftDocs/visualstudio-docs/issues/1550](https://github.com/MicrosoftDocs/visualstudio-docs/issues/1550)
[https://github.com/dotnet/roslyn/issues/30083](https://github.com/dotnet/roslyn/issues/30083)

---
In reply to: [219297113](https://github.com/dotnet/roslyn/pull/30043#discussion_r219297113) [](ancestors = 219297113,219241244)